docs/configuration/http_conn_man/route_config/rate_limits.rst

```diff
@@ -102,12 +105,25 @@ Remote Address
     "type": "remote_address"
   }
 
-The following descriptor is sent using the trusted address from :ref:`x-forwarded-for <config_http_conn_man_headers_x-forwarded-for>`:
+The following descriptor entry is appended to the descriptor and is populated using the trusted
+address from :ref:`x-forwarded-for <config_http_conn_man_headers_x-forwarded-for>`:
 
     * ("remote_address", "<:ref:`trusted address from x-forwarded-for <config_http_conn_man_headers_x-forwarded-for>`>")
 
-If *route_key* is set in the rate limit configuration, the following
-descriptor is sent as well:
+Route Key
```

Comment:
I think we should rename this at this point. You have made this fully generic (nice), so it's possible to compose multiple generic strings now. I would call it something like "generic_string" etc. (you can think of a good name).